apps/greencheck/management/commands/update_azure_ip_ranges.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import ipaddress
+import logging
+import json
+from apps.greencheck.models import GreencheckIp
+from apps.accounts.models import Hostingprovider
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+GREEN_REGIONS = (
+    ("Azure", "azure-code", settings.AZURE_PROVIDER_ID),
+)
+
+class MicrosoftCloudProvider:
+    def retrieve(self):
+        with open('apps/greencheck/management/commands/azure-ip-ranges.json') as json_file:
```

Comment:
Rather than using this string, would you please use the Pathlib for locating this file? It makes it cross platform, and the API is fairly nice to use. This documentation here shows some sample uses to copy, and explains why you might use it in more detail.

https://treyhunner.com/2019/01/no-really-pathlib-is-great/